cdc/processor/sinkmanager/mem_quota.go

```diff
@@ -156,6 +156,37 @@ func (m *memQuota) release(tableID model.TableID, resolved model.ResolvedTs) {
 	}
 }
 
+// clean all records of the table.
+func (m *memQuota) clean(tableID model.TableID) {
+	m.mu.Lock()
+	defer m.mu.Unlock()
+
+	if _, ok := m.tableMemory[tableID]; !ok {
+		// This can happen when the table has no data and never been recorded.
+		log.Warn("Table consumed memory records not found",
+			zap.String("namespace", m.changefeedID.Namespace),
+			zap.String("changefeed", m.changefeedID.ID),
+			zap.Int64("tableID", tableID))
+		return
+	}
+	cleaned := uint64(0)
+	records := m.tableMemory[tableID]
+	for _, record := range records {
+		cleaned += record.size
+	}
+	m.usedBytes -= cleaned
+	delete(m.tableMemory, tableID)
+	log.Debug("MemoryQuotaTracing: Clean table memory records",
```

Comment:
Seems other mem quota tracing logs are in callers and after the memory quota operation finished. How about make this functions returns `cleanedBytes`, and log the message in callers?

Reply:
Updated! Thanks! Good suggestion.